pkg/test/main_entry.go

```diff
@@ -38,21 +49,63 @@ func MainEntry(m *testing.M) {
 	globalManPath := flag.String(GlobalManPathFlag, "", "path to operator manifest")
 	namespacedManPath := flag.String(NamespacedManPathFlag, "", "path to rbac manifest")
 	singleNamespace = flag.Bool(SingleNamespaceFlag, false, "enable single namespace mode")
+	localOperator := flag.Bool(LocalOperatorFlag, false, "enable if operator is running locally (not in cluster)")
 	flag.Parse()
 	// go test always runs from the test directory; change to project root
 	err := os.Chdir(*projRoot)
 	if err != nil {
 		log.Fatalf("failed to change directory to project root: %v", err)
 	}
-	if err := setup(kubeconfigPath, namespacedManPath); err != nil {
+	if err := setup(kubeconfigPath, namespacedManPath, *localOperator); err != nil {
 		log.Fatalf("failed to set up framework: %v", err)
 	}
+	// setup local operator command, but don't start it yet
+	var localCmd *exec.Cmd
+	var localCmdOutBuf, localCmdErrBuf bytes.Buffer
+	if *localOperator {
+		// taken from commands/operator-sdk/cmd/up/local.go
+		runArgs := append([]string{"run"}, []string{filepath.Join(scaffold.ManagerDir, scaffold.CmdFile)}...)
```

Comment:
Not sure, if this works. I think, we need to get the ROOT of project and from there this path works.

Reply:
`main_entry.go` changes to the root project directory earlier in the code: https://github.com/operator-framework/operator-sdk/pull/781/files/782404032b6770eb86c8b58496a47c4d5d217e0b#diff-ebb4940df67225dc5f582034edb939ceR55

Reply:
Ahh I missed it. Thank you. I also wanted to point that from `projectRoot` which is test-framework, we need to have new cmd/manager.go which you added now for e2es to pass.